orangewidget/report/report.py

```diff
@@ -590,7 +598,22 @@ def get_html_img(scene):
     filename.open(QIODevice.WriteOnly)
     PngFormat.write(filename, scene)
     img_encoded = byte_array.toBase64().data().decode("utf-8")
-    return "<img src='data:image/png;base64,%s'/>" % img_encoded
+    return '<img {} src="data:image/png;base64,{}"/>'.format(
+        ("" if max_height is None
+         else 'style="max-height: {}px"'.format(max_height)),
+        img_encoded
+    )
+
+
+def get_icon_html(icon: QIcon, size: QSize) -> str:
+    print(size)
+    print(size.isValid())
+    byte_array = QByteArray()
+    buffer = QBuffer(byte_array)
+    buffer.open(QIODevice.WriteOnly)
+    icon.pixmap(size).save(buffer, "PNG")
+    img_encoded = byte_array.toBase64().data().decode("utf-8")
```

Comment:
Tests fail since this line returns an empty string. I tried to reproduce this behavior locally but I can't even when downgrading Pyqt to a version equal to the version that Tavis uses (I tested on python3.7). @ales-erjavec do you have any idea what is the problem here?

Reply:
Maybe
```diff
     buffer.open(QIODevice.WriteOnly)
     icon.pixmap(size).save(buffer, "PNG")
+    buffer.close()
     img_encoded = byte_array.toBase64().data().decode("utf-8")
```

Also ss `icon.pixmap(size)` valid?

Reply:
Thank you.